eng/pipelines/coreclr/templates/run-performance-job.yml

```diff
@@ -89,6 +89,8 @@ jobs:
         export ORIGPYPATH=$PYTHONPATH
         export CRYPTOGRAPHY_ALLOW_OPENSSL_102=true;
         echo "** Installing prerequistes **";
+        echo "** Waiting for dpkg to unlock (up to 2 minutes) **" &&
```

Comment:
We could use an if statement inside the while loop to only print this if dpkg is locked (see my other PR comment).

Reply:
Good idea, I have a run testing this out.